flink-staging/flink-scala-shell/src/main/scala/org/apache/flink/api/scala/FlinkILoop.scala

```diff
@@ -95,24 +140,49 @@ class FlinkILoop(
     new File(tmpDirBase, "scala_shell_commands.jar")
   }
 
-  private val packageImports = Seq[String](
-    "org.apache.flink.core.fs._",
-    "org.apache.flink.core.fs.local._",
-    "org.apache.flink.api.common.io._",
-    "org.apache.flink.api.common.aggregators._",
-    "org.apache.flink.api.common.accumulators._",
-    "org.apache.flink.api.common.distributions._",
-    "org.apache.flink.api.common.operators._",
-    "org.apache.flink.api.common.operators.base.JoinOperatorBase.JoinHint",
-    "org.apache.flink.api.common.functions._",
-    "org.apache.flink.api.java.io._",
-    "org.apache.flink.api.java.aggregation._",
-    "org.apache.flink.api.java.functions._",
-    "org.apache.flink.api.java.operators._",
-    "org.apache.flink.api.java.sampling._",
-    "org.apache.flink.api.scala._",
-    "org.apache.flink.api.scala.utils._"
-  )
+  private val packageImports =
+    streaming match {
+      case StreamingMode.BATCH_ONLY => Seq[String](
+      "org.apache.flink.core.fs._",
+      "org.apache.flink.core.fs.local._",
+      "org.apache.flink.api.common.io._",
+      "org.apache.flink.api.common.aggregators._",
+      "org.apache.flink.api.common.accumulators._",
+      "org.apache.flink.api.common.distributions._",
+      "org.apache.flink.api.common.operators._",
+      "org.apache.flink.api.common.operators.base.JoinOperatorBase.JoinHint",
+      "org.apache.flink.api.common.functions._",
+      "org.apache.flink.api.java.io._",
+      "org.apache.flink.api.java.aggregation._",
+      "org.apache.flink.api.java.functions._",
+      "org.apache.flink.api.java.operators._",
+      "org.apache.flink.api.java.sampling._",
+      "org.apache.flink.api.scala._",
+      "org.apache.flink.api.scala.utils._"
+    )
+      case StreamingMode.STREAMING => Seq[String](
+       "org.apache.flink.core.fs._",
+      "org.apache.flink.core.fs.local._",
+      "org.apache.flink.api.common.io._",
+      "org.apache.flink.api.common.aggregators._",
+      "org.apache.flink.api.common.accumulators._",
+      "org.apache.flink.api.common.distributions._",
+      "org.apache.flink.api.common.operators._",
+      "org.apache.flink.api.common.operators.base.JoinOperatorBase.JoinHint",
+      "org.apache.flink.api.common.functions._",
+      "org.apache.flink.api.java.io._",
+      "org.apache.flink.api.java.aggregation._",
+      "org.apache.flink.api.java.functions._",
+      "org.apache.flink.api.java.operators._",
+      "org.apache.flink.api.java.sampling._",
+      "org.apache.flink.api.scala._",
+      "org.apache.flink.api.scala.utils._",
+      "org.apache.flink.streaming._",
+      "org.apache.flink.streaming.connectors.rabbitmq._",
```

Comment:
This is copied from:
https://github.com/apache/flink/commit/c82ebbfce0b11a4b4de3126fb02ccfdad80e0837
just for the streaming case.